homepairs/homepairsUI/src/Elements/Buttons/ServiceProviderButton.tsx

```diff
@@ -72,21 +118,33 @@ export default function ServiceProviderButton(props: ServiceProviderButtonProps)
                 style={styles.buttonStyle}
                 onPress={() => onClick(serviceProvider.provId, serviceProvider.name)}
             >
-                <Text style={styles.titleText}>
-                    {serviceProvider.name}
-                </Text>
-                <Text style={styles.companyDetailsText}>
-                    {serviceProvider.skills}
-                </Text>
-                <Text style={styles.companyDetailsText}>
-                    {serviceProvider.email}
-                </Text>
+                <View style={{ flexDirection: 'row' }}>
+                    <View style={styles.leftColumnStyle}>
+                        <View style={styles.imageContainer}>
+                            {renderLogo(serviceProvider.name, serviceProvider.logo)}
+                        </View>
+                        <Text style={styles.payRateText}>
+                            ${serviceProvider.payRate} / hour
+                        </Text>
+                    </View>
+                    <View style={styles.rightColumnStyle}>
+                        <Text style={styles.titleText}>
+                            {serviceProvider.name}
+                        </Text>
+                        <Text style={styles.companyDetailsText}>
+                            <br />{serviceProvider.skills}
```

Comment:
Yea, I mixed and thought serviceProvider was a Component. However, the <br /> are html and will break the program on mobile devices

Reply:
Made the changes but next time I will simply make a change request.

Reply:
everything uses carrots / I guess I didn't realize it was html. Looking up stylesheet stuff gives me a lot of html/css results, usually its obvious enough to tell though

Reply:
feel free to make a change request if you're busy! definitely easy for me to overlook styling stuff / not as familiar